src/cli/commands/upgrade.js

```diff
@@ -27,16 +28,37 @@ export async function run(
     peerDependencies,
   } = await config.readRootManifest() || {};
   const allDependencies = Object.assign({}, peerDependencies, optionalDependencies, devDependencies, dependencies);
+  let addArgs = [];
```

Comment:
Regarding the global workflow of the code, I'd prefer using ES5 array functions:

```js
let addArgs = Object.keys(allDependencies);

if (flags.scope) {
    addArgs = addArgs.filter(arg => {
        return ...;
    });
}

addArgs = addArgs.map(arg => {
    return ...;
});
```